### PR TITLE
Enable console and debug

### PR DESCRIPTION
**Pull request by thaynaracsantos**
# Title
[feature] Enable console and debug

### Status

READY

### Whats?

Enable console and debug

### Why?

To improve the visualization of the debug messages

### How?

Changes were made in the logger factory constructor, to allow the setup and enable console, debug messages were implemented and tests were added.

### Definition of Done:
- [ ] Increases API documentation
- [ ] Implements integration tests
- [X] Implements unit tests
- [X] Is there appropriate logging included?
- [ ] Does this add new dependencies?
- [ ] Does need add new version in changelog?
- [ ] Does need update readme, contributing, etc?
- [ ] Does need change in CI server?
- [ ] Will this feature require a new piece of infrastructure be implemented?
- [ ] Does this PR require a blog post? If so, ensure marketing has signed off on content.

### CODE REVIEW

File: Serilog.Builder/Factory/LoggerFactory.cs [hunk 1]
```diff
@@ -22,22 +23,33 @@ public class LoggerFactory : ILoggerFactory
...
                 .SetupSeq(seqOptions.Value)
                 .SetupSplunk(splunkOptions.Value)
                 .SetupGoogleCloudLogging(gcpOptions.Value)
                 .BuildLogger();
 
             this._loggerDefault = new LoggerDefault(logger);
+
+            if (loggerOptions.Value.DebugEnable)
+            {
+                builder.EnableConsole();
```
Comment (ThiagoBarradas) [inline Serilog.Builder/Factory/LoggerFactory.cs]: EnableDebug

File: Serilog.Builder/Factory/LoggerFactory.cs [hunk 2]
```diff
@@ -22,22 +23,33 @@ public class LoggerFactory : ILoggerFactory
...
+            }
+
+            logger.Debug($"Logger working");
+            _loggerDefault.DebugAsync($"LoggerDefault working");
```
Comment (ThiagoBarradas) [inline Serilog.Builder/Factory/LoggerFactory.cs]: Insert this in previous if

Comment (thaynaracsantos) [reply Serilog.Builder/Factory/LoggerFactory.cs]: Done

File: Serilog.Builder/Factory/LoggerFactory.cs (hunk 1 above)
Comment (thaynaracsantos) [reply Serilog.Builder/Factory/LoggerFactory.cs]: Done

**Review by ThiagoBarradas — APPROVED** [review]

**Pull request merged**